common/djangoapps/cme_registration/models.py

```diff
@@ -54,50 +54,37 @@ class Meta:
 
-    SPECIALTY_CHOICES = (('Addiction_Medicine', 'Addiction Medicine'),
-                         ('Adolescent_Medicine', 'Adolescent Medicine'),
-                         ('Allergy', 'Allergy'),
-                         ('Anesthesiology', 'Anesthesiology'),
-                         ('Cardiology', 'Cardiology'),
-                         ('Complimentary_Medicine', 'Complimentary Medicine'),
-                         ('Critical_Care_Medicine_&_ICU', 'Critical Care Medicine & ICU'),
-                         ('Dentistry', 'Dentistry'),
-                         ('Dermatology', 'Dermatology'),
-                         ('Emergency_Medicine', 'Emergency Medicine'),
-                         ('Endocrinology', 'Endocrinology'),
-                         ('Family_Practice', 'Family Practice'),
-                         ('Gastroenterology_&_Hepatology', 'Gastroenterology & Hepatology'),
-                         ('General_Practice', 'General Practice'),
-                         ('Gerontology', 'Gerontology'),
-                         ('Head_&_Neck_Surgery', 'Head & Neck Surgery'),
-                         ('Health_Education', 'Health Education'),
-                         ('Hematology', 'Hematology'),
-                         ('Immunology_&_Rheumatology', 'Immunology & Rheumatology'),
-                         ('Infectious_Disease', 'Infectious Disease'),
-                         ('Internal_Medicine', 'Internal Medicine'),
-                         ('Neonatology', 'Neonatology'),
-                         ('Nephrology', 'Nephrology'),
-                         ('Neurology', 'Neurology'),
-                         ('Neurosurgery', 'Neurosurgery'),
-                         ('Nutrition', 'Nutrition'),
-                         ('Obstetrics_&_Gynecology', 'Obstetrics & Gynecology'),
-                         ('Oncology', 'Oncology'),
-                         ('Ophthalmology', 'Ophthalmology'),
-                         ('Orthopaedic_Surgery', 'Orthopaedic Surgery'),
-                         ('Palliative_Care', 'Palliative Care'),
-                         ('Pathology', 'Pathology'),
-                         ('Pediatrics', 'Pediatrics'),
-                         ('Pharmacology', 'Pharmacology'),
-                         ('Physical_Medicine_&_Rehabilitation', 'Physical Medicine & Rehabilitation'),
-                         ('Psychiatry', 'Psychiatry'),
-                         ('Psychology', 'Psychology'),
-                         ('Public_Health', 'Public Health'),
-                         ('Pulmonology', 'Pulmonology'),
-                         ('Radiation_Oncology', 'Radiation Oncology'),
-                         ('Radiology', 'Radiology'),
-                         ('Surgery', 'Surgery'),
-                         ('Transplant', 'Transplant'),
-                         ('Urology', 'Urology'))
+    SPECIALTY_CHOICES = (('Allergy,_Immunology,_&_Rheumatology','Allergy, Immunology, & Rheumatology'),
```

Comment:
@karandikar Since we're modifying these lines too, let's also reformat following the same convention we used below (whitespace/alignment).